pkg/crc/machine/start.go

```diff
@@ -507,39 +506,27 @@ func updateSSHKeyAndCopyKubeconfig(sshRunner *crcssh.Runner, name string, crcBun
 	return nil
 }
 
-func configProxyForCluster(ocConfig oc.Config, sshRunner *crcssh.Runner, sd *systemd.Commander, proxy *network.ProxyConfig, instanceIP string) (err error) {
+func ensureKubeletAndCRIOAreConfiguredForProxy(sshRunner *crcssh.Runner, proxy *network.ProxyConfig, instanceIP string) (err error) {
```

Comment:
We don't stop the crio service by default so it will be running as soon as VM is up so either that is handled in snc side or here.

Reply:
crio is disabled by default and it is started only when we start kubelet.

```
[root@crc-rlc2h-master-0 core]# systemctl status crio
● crio.service - Open Container Initiative Daemon
   Loaded: loaded (/usr/lib/systemd/system/crio.service; disabled; vendor preset: disabled)
[root@crc-rlc2h-master-0 core]# systemctl list-dependencies kubelet
kubelet.service
● ├─crio.service
● ├─machine-config-daemon-firstboot.service
● ├─rpc-statd.service

```

Reply:
Ah in that case I am OK